guides/mn-application-aws-lambda-graalvm/mn-application-aws-lambda-graalvm.adoc

```diff
@@ -16,13 +16,17 @@ common:completesolution.adoc[]
 
 common:create-app-features.adoc[]
 
+== Controller
+
 The generated application contains a `HomeController`. It responds to POST request to `/`.
 
 source:HomeController[]
 
 callout:controller[arg0=/]
 * The `@Get` annotation maps HTTP requests to `/` to the `index` method.
 
+== Controller Test
+
```

Comment:
This guide has a screenshot that shows it taking 1s for a Graal native lambda to be executed...

I assume this is wrong?

Reply:
No, it was what I got with a cold startup and 128Mb of memory. I have changed it use the warm startup screenshot instead